js/foundation.offcanvas.js

```diff
@@ -282,7 +309,7 @@ class OffCanvas extends Plugin {
    * @fires OffCanvas#opened
    */
   open(event, trigger) {
-    if (this.$element.hasClass('is-open') || this.isRevealed) { return; }
+    if (this.$element.hasClass('is-open') || this.isRevealed || this.isInCanvas) { return; }
```

Comment:
hmm Why do we rely on `isInCanvas` or responsive classes instead of calling `.open()` and relying on `.is-open`

Reply:
I had some trouble to understand the role of `isInCanvas` here, so 3 weeks later it's even worst :D. I'll try to take a fresh look at this tomorrow.

Reply:
@ncoden have you been able to take a look at this last requested change?
Would like to see this PR merged for 6.5.0 :-)